example/lib/screens/surface/tiles.dart

```diff
@@ -82,40 +90,43 @@ ListView.builder(
         ),
         CardHighlight(
           child: Row(crossAxisAlignment: CrossAxisAlignment.start, children: [
-            Container(
-              height: 400,
-              width: 350,
```

Comment:
Again, I think a `ConstrainedBox` with `maxWidth` of `350.0` fits it better here

Reply:
I've tried this approach, but it didn't work as expected. No responsiveness.
If you have a better solution just give me a hint.

Reply:
Again, the same idea.